drivers/scheduler/k8s/k8s.go

```diff
@@ -2088,44 +2097,46 @@ func (k *K8s) getPodsForApp(ctx *scheduler.Context) ([]v1.Pod, error) {
 
 //Describe describe the test case
 func (k *K8s) Describe(ctx *scheduler.Context) (string, error) {
-	k8sOps := k8s_ops.Instance()
+	k8sOps := k8sops.Instance()
 	var buf bytes.Buffer
 	var err error
-	for _, spec := range ctx.App.SpecList {
-		if obj, ok := spec.(*apps_api.Deployment); ok {
-			buf.WriteString(insertLineBreak(obj.Name))
-			var depStatus *apps_api.DeploymentStatus
+	for _, specObj := range ctx.App.SpecList {
+		if obj, ok := specObj.(*appsapi.Deployment); ok {
+			buf.WriteString(insertLineBreak(fmt.Sprintf("Deployment: %s", obj.Name)))
```

Comment:
add the namespace here. Do the same at other places.

Something like

`fmt.Sprintf("Deployment: [%s] %s", obj.Namespace, obj.Name))`